tests/models/test_DurableOrchestrationClient.py

```diff
@@ -610,3 +610,113 @@ async def test_rewind_with_no_rpc_endpoint(binding_string):
         await client.rewind(INSTANCE_ID, REASON)
     ex_message = str(ex.value)
     assert ex_message == expected_exception_str
+
```

Comment:
I added these tests by emulating the tests like terminate/wait_for_completion: test different returns from the http request. But I am not sure if these are too duplicate. Let me know if you have any thoughts about this. Thanks!

Reply:
seems fine to me :)